README.mdown

```diff
@@ -39,6 +41,30 @@ curl -X POST -H "Content-Type: application/json" \
      http://127.0.0.1:8080/incoming/your-secret-token
 ```
 
+### UptimeRobot
+
+The URL is constructed as follows: `http(s)://<host>:<port>/incoming/uptimerobot/<your-secret-token>`
+
+#### Usage
+
+Send a GET request to the webhook URL with the follow query params.
+In UptimeRobot's settings, add the following webhook (the trailing ampersand
+is required):
+
+```
+http(s)://<host>:<port>/incoming/uptimerobot/<your-secret-token>?room=ops@example.com&
+```
+
+| Key | Value |
+| --- | ----- |
+| room | (string) The room/channel you want to post to. |
```

Comment:
This does not make sense to me, as it's simply copied from the existing README without actually applying here. There's no body for adding keys and values, and if there's only a single param that can be added to customize the hook, then I don't think we need a table to explain it.